src/main/resources/sam.conf

```diff
@@ -3,6 +3,9 @@ akka {
     server {
       idle-timeout = 180 s
       request-timeout = 60 s
+      parsing {
+        ignore-illegal-header-for = ${?IGNORE_ILLEGAL_HEADERS}
```

Comment:
mind adding a comment explaining what this is and why it might be used?

Reply:
gladly! hows that?